Engine/platform/osx/acplmac.cpp

```diff
@@ -113,6 +151,31 @@ void AGSMac::ShutdownCDPlayer() {
   //cd_exit();
 }
 
+const char* AGSMac::GetAllUsersDataDirectory()
+{
+  return commonDataPath;
+}
+
+const char *AGSMac::GetUserSavedgamesDirectory()
+{
+  return libraryApplicationSupport;
+}
+
+const char *AGSMac::GetUserConfigDirectory()
+{
+  return libraryApplicationSupport;
+}
+
+const char *AGSMac::GetAppOutputDirectory()
+{
+  return commonDataPath;
+}
+
+const char *AGSMac::GetIllegalFileChars()
+{
+  return "\\/:?\"<>|*"; // keep same as Windows so we can sync.
```

Comment:
I think we need elaboration on why this has to be synced with Windows.
I suspect there may be misunderstanding in how this function is used, because I do not see any immediate reasons to replicate Windows behavior here.